src/marqo/tensor_search/models/index_info.py

```diff
@@ -36,6 +58,15 @@ def get_text_properties(self) -> dict:
         bool fields.
 
         """
+        # get_text_properties will flatten the object properties
+        # Example: left-text_properties   right-true_text_properties
+        # {'Description': {'type': 'text'},                                                         {'Description': {'type': 'text'},
+        #  'Genre': {'type': 'text'},                                                                'Genre': {'type': 'text'},
+        #  'Title': {'type': 'text'},                                                                'Title': {'type': 'text'},
+        #  'my_combination_field': {'properties': {'lexical_field': {'type': 'text'}, ----->         'my_combination_field.lexical_field': {'type': 'text'},
+        #                                          'my_image': {'type': 'text'},                     'my_combination_field.my_image': {'type': 'text'},
+        #                                          'some_text': {'type': 'text'}}}}                   'my_combination_field.some_text': {'type': 'text'}}
+
```

Comment:
nit: may as well make this part of the docstring at this stage
note: we should rename this function at some point

Reply:
done adding to docstring